docs/book/src/developer/tilt.md

```diff
@@ -55,6 +55,9 @@ If you prefer JSON, you can create a `tilt-settings.json` file instead. YAML wil
 
 </aside>
 
+### Using a local image registry
```

Comment:
The title and the suggested command are misleading. The command `make kind-cluster` actually creates the kind cluster (along with the local registry, not just the local registry.).

Also, you don't need to specify `default_registry: localhost:5000` in your tilt-settings.yaml for it to pick up the local registry. Tilt will automatically pick up the local registry even is `default_registry` is pointing to something else. From tilt logs: `Default registry specified, but will be ignored in favor of auto-detected registry.`

Maybe instead of adding a separate section what about adding a note under the "Create a kind cluster" section under the `./hack/kind-install-for-capd.sh` command that says that this command automatically create a local registry and tilt will pick it up for faster image pulls?